test/converters/xcresult3/action_test_plan_summary.go

```diff
@@ -53,14 +54,15 @@ func (s ActionTestPlanRunSummaries) tests() map[string][]ActionTestSummaryGroup
 			}
 
 			summaryGroupsByName[name] = tests
+			testSuiteOrdering = append(testSuiteOrdering, name)
```

Comment:
`summaryGroupsByName[name] = tests` will keep the last tests with the same name,
while `testSuiteOrdering = append(testSuiteOrdering, name)` will store all the names (even the same name multiple times), shouldn't we use:

```
			if _, found := summaryGroupsByName[name]; !found {
				testSuiteOrdering = append(testSuiteOrdering, name)
			}
			summaryGroupsByName[name] = tests
```